File: Selenium/TP4/money_js/src/money.js
var money= (function(){
	function money(v,curr) {
		this.v=v;
		this.curr=curr;
	}

	money.prototype.getCurrency=function () {
		return this.curr;
	}
	money.prototype.getValue=function () {
		return this.v;
	}
	money.prototype.equals=function (otherM) {
		return (otherM.getValue()==this.getValue()) ;
	}
	money.prototype.toString=function toString() {
		return this.getValue()+" ("+this.getCurrency()+")" ;
	}


return money;
})();

File: TP1/money_src/src/fr/ulille1/fil/odeva/Money.java
package  fr.ulille1.fil.odeva;

import java.util.HashMap;
import java.util.Map;

public class Money {
	private int value;
	private String currency;
	

	Money(int value, String currency)
	{
		this.value=value;
		this.currency=currency;
	}

	public int getValue()
	{
		return this.value;
	}

	public String getCurrency()
	{
		return this.currency;
	}

    public String toString() {
     	return this.getValue()+" ("+this.getCurrency()+")";
    }
    
    
    public boolean equals(Object o) {
		if(o instanceof Money){
			Money money = (Money)o;
			return this.getValue() == money.getValue() && this.getCurrency() == money.getCurrency();
		}
		else
    	return false;
    }

}
